File: MPI/mpicheck/tests/functions/2.0/MPI_Get_address.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const void *var_0;
    MPI_Aint *var_1;
    int ret;
    /* calls */
    ret = MPI_Get_address(var_0, var_1);
    ret = PMPI_Get_address(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Get_address.f90

        
        program main
        use mpi
        
        TYPE(INTEGER) var_0(10)
       INTEGER(KIND=MPI_ADDRESS_KIND) var_1
       INTEGER var_2
        call mpi_get_address(var_0, var_1, var_2)
       call pmpi_get_address(var_0, var_1, var_2)
        end program main
    